CMakeLists.txt

```diff
@@ -156,6 +156,7 @@ message(STATUS "Build type: " ${CMAKE_BUILD_TYPE})
 # Static/Shared library
 # ---------------------------------------------------------------------------------------
 set(SPDLOG_SRCS src/spdlog.cpp src/stdout_sinks.cpp src/color_sinks.cpp src/file_sinks.cpp src/async.cpp src/cfg.cpp)
+list(APPEND SPDLOG_SRCS include/spdlog/sinks/ansicolor_sink-inl.h)
```

Comment:
Should be included in https://github.com/gabime/spdlog/blob/v1.x/src/stdout_sinks.cpp instead